airflow/models/mappedoperator.py

```diff
@@ -201,7 +201,7 @@ def _expand(self, **mapped_kwargs: "Mappable") -> "MappedOperator":
         from airflow.operators.empty import EmptyOperator
 
         validate_mapping_kwargs(self.operator_class, "expand", mapped_kwargs)
-        prevent_duplicates(self.kwargs, mapped_kwargs, fail_reason="mapping already partial")
+        prevent_duplicates(self.kwargs, mapped_kwargs, fail_reason="unmappable or already specified")
```

Comment:
```suggestion
        prevent_duplicates(self.kwargs, mapped_kwargs, fail_reason="unmappable or already specified.")
```
Maybe a dot here? The error from the issue is difficult to read

Reply:
The error formatting code in `prevent_duplicates` puts a colon (`:`) after this message, so if we want to add a period here we need to redo the formatting.

This currently formats into

```
TypeError: unmappable or already specified argument: params
```

and a dot would be out of place.